src/renderer/components/configure-recording-set-page/index.tsx

```diff
@@ -249,6 +252,11 @@ const ConfigureRecordingSetPage: FC<{
           </Row>
         </Col>
       </Container>
+      <Positional position={'top-right'}>
+        <Button variant={'outline-secondary'} style={{ minWidth: '120px' }} onClick={onSelect}>
+          {'Setting'}
```

Comment:
Is this a button using an icon or a button with just text "Setting"?

Reply:
change to an image icon